src/app/static/src/tests/components/modelCalibrate/modelCalibrate.test.ts

```diff
@@ -122,6 +122,18 @@ describe("Model calibrate component", () => {
         expect(wrapper.find("button").classes()).not.toContain("btn-submit");
     });
 
+    it("renders string progress message", () => {
```

Comment:
Do you need to add a test for when it should be rendering 'Calibrating...'

Reply:
THere's actually a test already, but I'll rename it make it clear it only applies when no progress information is available